packages/create-pantheon-decoupled-kit/src/templates/next-drupal-app/lib/stores.ts

```diff
@@ -0,0 +1,59 @@
+import { DRUPAL_URL } from './constants';
+import { DrupalState } from '@pantheon-systems/drupal-kit';
+
+// For each locale, make an instance of DrupalState (LocaleStore)
+export const makeLocaleStores = ({ locales, debug = false } : {locales: string[], debug: boolean}) =>
+	locales.length > 1
+		? locales.map(
+				(locale) =>
+					new DrupalState({
+						apiBase: DRUPAL_URL,
+						defaultLocale: locale,
+						debug: debug,
+						clientId: process.env.CLIENT_ID,
+						clientSecret: process.env.CLIENT_SECRET,
+					}),
+		  )
+		: [
+				new DrupalState({
+					apiBase: DRUPAL_URL,
+					debug: debug,
+					clientId: process.env.CLIENT_ID,
+					clientSecret: process.env.CLIENT_SECRET,
+				}),
+		  ];
+
+// exporting so we can reuse in
+// getStaticProps and getServerSideProps
+/**
+ * @type {DrupalState[]}
+ * @remarks These stores have auth enabled. Authenticated and anonymous request control takes place on the level of the request.
+ * @see {@link https://project.pages.drupalcode.org/drupal_state/en/getting-objects/} for more information.
+ *
+ */
+export const globalDrupalStateStores = makeLocaleStores({
+	locales: process.env.locales as unknown as string[],
+	debug: process.env.DEBUG_MODE as unknown as boolean || false,
```

Comment:
When converting this file to TS I landed this typing `process.env.DEBUG_MODE as unknown as boolean`. I feel like it smells a bit weird though, lmk if you have any thoughts.